tensorflow/stream_executor/rocm/rocm_blas.cc

```diff
@@ -2530,13 +1825,6 @@ bool ROCMBlas::DoBlasGemmBatched(
     float beta, const port::ArraySlice<DeviceMemory<Eigen::half> *> &c,
     int ldc, int batch_count, ScratchAllocator *scratch_allocator) {
   return false;
```

Comment:
should there be a LOG(ERROR) added here?

Reply:
will fix.